airbyte-cdk/java/airbyte-cdk/dependencies/src/testFixtures/kotlin/io/airbyte/workers/general/DbtTransformationRunner.kt

```diff
@@ -119,25 +119,24 @@ class DbtTransformationRunner(
                     emptyMap(),
                     *dbtArguments.toTypedArray<String>()
                 )
+            this.process = process
             LineGobbler.gobble(
-                process!!.inputStream,
+                process.inputStream,
                 { msg: String? -> LOGGER.info(msg) },
                 CONTAINER_LOG_MDC_BUILDER
             )
             LineGobbler.gobble(
-                process!!.errorStream,
+                process.errorStream,
                 { msg: String? -> LOGGER.error(msg) },
                 CONTAINER_LOG_MDC_BUILDER
             )
 
             TestHarnessUtils.wait(process)
 
-            return process!!.exitValue() == 0
+            return process.exitValue() == 0
         } catch (e: Exception) {
             // make sure we kill the process on failure to avoid zombies.
-            if (process != null) {
-                TestHarnessUtils.cancelProcess(process)
-            }
+            process?.let { TestHarnessUtils.cancelProcess(process) }
```

Comment:
This `?` works now that `process` is `lateinit` a non-nullable type? Surprising.